GOFevaluation/utils.py

```diff
@@ -407,10 +426,17 @@ def plot_equiprobable_histogram(data_sample, bin_edges, order=None,
                 j += 1
             i += 1
 
-    fig = mpl.pyplot.gcf()
+    ax.set_xlim(xlim)
+    ax.set_ylim(ylim)
+
+    if draw_colorbar:
+        fig = mpl.pyplot.gcf()
 
-    fig.colorbar(mpl.cm.ScalarMappable(norm=norm, cmap=cmap), ax=ax,
-                 label=label)
+        fig.colorbar(
+            mpl.cm.ScalarMappable(norm=norm, cmap=cmap),
+            ax=ax,
+            label=label,
+        )
```

Comment:
I think we should now set the `extend` kwarg if the values are out-of-bounds due to the hard-set vmin and vmax.
Maybe something like (I'm sure there is much more efficient code for this..):
```
extend = 'neither'
if norm.vmin > min(n) & norm.vmax < max(n):
    extend='both'
elif norm.vmin > min(n):
    extend = 'min'
elif norm.vmax < max(n):
    extend = 'max'    
```
and then pass `extend` to `fig.colorbar`..